src/components-examples/cdk/text-field/index.ts

```diff
@@ -1,6 +1,7 @@
 import {TextFieldModule} from '@angular/cdk/text-field';
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
+import {MatButtonModule} from '@angular/material/Button';
```

Comment:
This should be lower-case `button`.

```suggestion
import {MatButtonModule} from '@angular/material/button';
```